src/components/AdditionalMaterials/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { HelpCircle } from 'react-feather';
+import { Styles } from '../../constants';
+import { SbNavLink, SbNavlinkProps } from '../SbNavLink';
+
+export interface AdditionalMaterialsProps {
```

Comment:
Is this interface used anywhere in this file? I cant see where `link` is being used

Reply:
This was put in for future use when the links will actually need to go somewhere.